scale-decode/src/visitor/mod.rs

```diff
@@ -1043,20 +1071,21 @@ mod test {
         impl Visitor for BailOutVisitor {
             type Value<'scale, 'info> = (&'scale [u8], u32);
             type Error = DecodeError;
+            type TypeResolver = PortableRegistry;
```

Comment:
For these tests we don't care about being generic over the resolver; setting it to `PortableRegistry` just makes everything basically the same as it was before (except that type IDs are now references in the signatures below).